### DISCUSSION TITLE
Passwd password change and login

### DISCUSSION THREAD

**Discussion by dmnitro** [telekom-security/tpotce#1071] [Q&A]
I was able to see results before but now it stays on the screen anyone happen to know why?
![image](https://user-images.githubusercontent.com/77514529/163568402-b141251b-11c1-4c98-8edf-005018a20c74.png)

**Comment by dmnitro** [comment]
I also can't seem to become root when i earlier could i can change the password using the password im trying to log in with but still can't use the new one. Very weird 
![image](https://user-images.githubusercontent.com/77514529/163568609-248ab606-2b54-4e48-b675-febfb9881543.png)
![image](https://user-images.githubusercontent.com/77514529/163568645-6d18fdb9-ec49-4d5e-a852-3e81392d5881.png)

**Reply by t3chn0m4g3** [reply]
`sudo su -`

**Reply by dmnitro** [reply]
Thank you do very much you know by  any chance what is wrong with the web UI? Only reason i can think of is that i created the Tpot yesterday at my other home so my IP is different.

**Reply by t3chn0m4g3** [reply]
Sorry, no idea.
